pages/linux/arecord.md

```diff
@@ -0,0 +1,11 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
```

Comment:
I couldn't find a specific page for this command, but the ALSA home page is a wiki, and the only page with references to this command that I could find was this: https://www.alsa-project.org/wiki/Asoundrc.
@sbrl, do you think we should add a link to this specific page or the wiki home?

Reply:
Could also link to the manpage at https://linux.die.net/man/1/arecord ?

Reply:
Added that in 90efa6b91c29687a0b12cf21907b644139d1e958

Reply:
Yeah, the man page sounds like a better idea in this case.